connectors/cli.py

```diff
@@ -90,11 +89,14 @@ def _parser():
 
 def run(args):
     """Runner"""
+
+    # load config
+    Config.load(args.config_file)
```

Comment:
let's use a real instance there,

```
config = Config(args.config_file)
```

Reply:
I want to make it a singleton and let the class variable `_yaml` hold the data, so it only loads once and is accessible globally(this doesn't mean it will, e.g. `ESClient` will still take config as a param instead of adding a dependency on `Config` class).